### ISSUE TITLE
Please bring back the zoom animation

### ISSUE THREAD

**Issue by nevster** [Quicksaver/Tab-Groups#290]
What makes the tab groups addon great is the ability to rely on visual locational awareness and memory. If it's just a list of things, it's not much better than just using bookmarks.

Now when, I click on the tab group icon, I can't tell which group I've been looking at - there's no visual indication. If it's too hard to do the zoom animation, at least add something like a brief flash of the group you've just 'minimised'

**Comment by Quicksaver** [comment]
> there's no visual indication

The active tab should have a darker tone, clearly distinguishable from the rest of the group(s) and background. The active group itself also has an extra highlighted "border", although I can understand it's not the most obvious thing to notice. But still, there are these visual indicators.

As explained in the release notes, the zoom animation was sloppy, would only look well under very specific circumstances, and they simply didn't work anymore with the new interface. Removing them was necessary, they may be replaced with other animations at a later time, but that may not happen soon at least.

**Comment by ForSerious** [comment]
Yes it was sloppy. I wouldn't care if it didn't animate at all. I just liked how it was set up; it was all visual.
How do I change back to an old version?

**Comment by Quicksaver** [comment]
> I just liked how it was set up

How the animation was set up? I don't understand this.

**Comment by ForSerious** [comment]
As in: When the TabGroups button was pressed, it would zoom out to all of the tabs in their respective groups. Now it just shows a wimpy little drop down menu. The closest I have found to the way it was, is I have to press 'manage my groups.' If I could changes that to the default action and have the icon changed back so that it's not the name of the current group, I probably wouldn't notice anything had changed. That would be the best.

**Comment by Quicksaver** [comment]
@ForSerious that has nothing to do with the zoom animation, if you still have any trouble after this or need to further discuss this, please open a new issue. As explained in the release notes, in the add-on description, and in the "How to use" page in the add-on's preferences, you can easily restore any previous behavior you want. You can disable the "Click to show the quick access panel" option in the add-on's preferences, so that clicking the button takes you directly to groups view, and you can still use the previous free arrange mode if you prefer it over the new interface.

**Comment by ForSerious** [comment]
My sorries. I just had a windows® user moment where the options in the upgrade were forced upon me and I didn't take enough time to inform myself of how to correct things.

**Comment by Rjs37** [comment]
I would probably suggest explaining this in the notes for the update. I actually thought the addon was broken when it first got applied due it showing the group name rather than the icon. I'm glad you've provided settings to change it back to similar to how it used (disable quick access, get rid of the group name and free arrange) to be, but I feel that they should probably be the default options / made much clearer how to revert back to what people are used to. 

For example I was looking through the settings trying to change the group view for quite a while before I came across the free arrange button on the tab listing screen.

**Comment by Quicksaver** [comment]
@Rjs37 this is not the correct place to have this discussion. If you still feel like you need to continue the discussion after this, please open a new issue or send me an e-mail. All of what you mentioned is explained in the release notes, in the add-on homepage description at AMO, and in the "How to use" page. As for changing the default behavior, I change default behavior when I feel it will benefit the majority of users, and especially new users, which is the case. I provided the ability to have the previous behavior for those who still prefer it, but other than that, whether a feature is new or already exists should not play any part in what becomes default. Something shouldn't be default forever just because it's what came first.

**Comment by nevster** [comment]
Ah right, yes, I can see the border now - it's way too subtle I think. Many of my tab groups have a lot of tabs in them such that they show in a stack, so that's the only indicator I have. Could you please change it to make it bolder? I'll raise a new issue.

I'd still like the zoom animation but understand the difficulty in doing this kind of thing. Thanks for your efforts!
